partner_document/models/partner_document_type.py

```diff
@@ -26,6 +26,19 @@ class PartnerDocumentType(models.Model):
         default=False,
         help="Check this if the document type does not require an expiration date.",
     )
+    classification_ids = fields.Many2many(
+        comodel_name="partner.classification",
+        compute="_compute_classification_ids",
+        store=False,
+        readonly=True,
+        string="Classifications",
+    )
+
+    def _compute_classification_ids(self):
+        for rec in self:
+            rec.classification_ids = rec.env["partner.classification"].search(
+                [("document_type_ids", "in", rec.id)]
+            )
```

Comment:
```suggestion
    classification_ids = fields.Many2many(
        comodel_name="partner.classification",
        relation="partner_classification_document_type_rel",
        column1="document_type_id",
        column2="classification_id",        
    )
```
This should work and it's a much eficient and elegant solution